legacy/clang_11/Dockerfile

```diff
@@ -55,6 +40,14 @@ RUN dpkg --add-architecture i386 \
        libncurses5-dev \
        libncursesw5-dev \
        liblzma-dev \
+    && wget https://apt.llvm.org/llvm.sh \
```

Comment:
that's super unsecure to run script downloaded from the intertet
at least checking sha of the file would be good

Reply:
It comes directly from llvm, but still, I had a certificate error on Mac. I'll collect what it's doing and push to the Dockerfile directly (it's updating apt files only). wget llvm was not a good idea, even as recommended the by official website here https://apt.llvm.org/

Reply:
Done, now it uses the regular approach